packages/cozy-harvest-lib/src/components/NewAccountModal.jsx

```diff
@@ -48,6 +48,9 @@ const NewAccountModal = ({ konnector, onSuccess, onDismiss }) => {
     }
 
     const accountId = triggersModel.getAccountId(trigger)
+    if (!accountId) {
```

Comment:
We're in a `handleSuccess` method of the `NewAccountModal`. How can we call it without having an `account`?

Reply:
It can happen with clisk konnectors when the flagship application navigates to the account before the realtime event with the account creation.
That is why the flagship application passes the account id with the loginSuccess message. And I forgot to use it. This is now fixed.

Reply:
Commit to remove?

Reply:
Same here...

Reply:
done